presto-main/src/main/java/com/facebook/presto/sql/planner/TranslationMap.java

```diff
@@ -133,10 +133,6 @@ else if (expressionToExpressions.containsKey(node)) {
 
     private Expression getMapping(Expression expression)
     {
-        if (!expressionToExpressions.containsKey(expression)) {
```

Comment:
So maybe callers should not verify this before invoking this method instead?

Reply:
You could retain this as `verify()`. It's always more readable than NPE when something goes incorrect.

Anyway, not very important, as this method apparently goes away later (so you could also drop this commit)